README.adoc

```diff
@@ -337,17 +482,40 @@ inventory-deployment-cf8f564c6-nctcr   1/1       Running   0          8m
 
 == Testing the microservices
 
+// Static guide instruction
+ifndef::cloud-hosted[]
 Run the tests by running the following command and appropriately substituting `[hostname]` 
 for the correct value.
 
 [role=command]
 ```
 mvn failsafe:integration-test -Dcluster.ip=[hostname]
 ```
+endif::[]
 
-A few tests are included for you to test the basic functions of the microservices. If a test failure
-occurs, then you might have introduced a bug into the code. To run the tests, wait for all pods to be
-in the ready state before proceeding further.
+// Cloud hosted guide instruction
+ifdef::cloud-hosted[]
+Run the following commands to store the proxy path of the **system** and **inventory** services.
+```
+cd /home/project/guide-kubernetes-microprofile-health/start
+NAMESPACE_NAME=`bx cr namespace-list | grep sn-labs- | sed 's/ //g'`
+SYSTEM_PROXY=localhost:8001/api/v1/namespaces/$NAMESPACE_NAME/services/system-service/proxy
+INVENTORY_PROXY=localhost:8001/api/v1/namespaces/$NAMESPACE_NAME/services/inventory-service/proxy
+```
+{: codeblock}
+
+Run the integration tests by using the following command:
+```
+mvn failsafe:integration-test \
+    -Dsystem.service.root=$SYSTEM_PROXY \
+    -Dinventory.service.root=$INVENTORY_PROXY
+```
+{: codeblock}
+endif::[]
+
+A few tests are included for you to test the basic functions of the microservices.
+If a test fails, then you might have introduced a bug into the code.
+To run the tests, wait for all pods to be in the ready state before proceeding further.
```

Comment:
To run the tests, wait for all pods to be in the ready state before proceeding further.
--->
Wait for all pods to be in the ready state before you run the tests.